src/sentry/api/endpoints/organization_member_index.py

```diff
@@ -159,10 +159,18 @@ def get(self, request, organization):
                 else:
                     queryset = queryset.none()
 
+        is_detailed = request.GET.get("detailed", "0") != "0"
```

Comment:
@NisanthanNanthakumar I thought we wanted something like:
```
is_detailed = request.GET.get("expand") == "1"
```
based on what Dan said?

Reply:
@scefali from discussion: https://github.com/getsentry/sentry/pull/23935#issuecomment-781763424
I implemented it in https://github.com/getsentry/sentry/pull/23935

Reply:
oh cool! I get what expand does now